tests/compiler.js

```diff
@@ -396,11 +398,26 @@ if (args.parallel && coreCount > 1) {
     process.send({ cmd: "ready" });
   } else {
     const tests = getTests();
+    for (let name of [ // Run tests known to take exceptionally long first
+      "std/libm",
```

Comment:
probably much simpler just get this items from sorted list by source file sizes?

Reply:
Just tried this, but unfortunately doesn't work well for `std/libm`, `runtime-full` and `runtime-stub` which are small but take long. Overall ordering the tests feels overoptimized ^^